.travis.yml

```diff
@@ -0,0 +1,9 @@
+osx_image: xcode7.2
+language: objective-c
```

Comment:
`language: objective-c`
This line is optional somehow.
https://blog.travis-ci.com/2015-12-10-xcode-72-is-ready